setup/action.yml

```diff
@@ -23,6 +23,9 @@ inputs:
   stack-setup-ghc:
     required: false
     description: 'If specified, enable-stack must be set. Will run stack setup to install the specified GHC'
+  cache:
+    required: false
+    description: 'If specified, automatically caches the cabal-related folders.'
```

Comment:
A catch-all cache option should work for all the build tools the action supports, so having the description reflect that would be less confusing.